internal/ecies.go

```diff
@@ -36,8 +36,7 @@ func (ec *EciesCrypto) Decrypt(value string) ([]byte, error) {
 }
 
 func (ec *EciesCrypto) Encrypt(value string) (string, error) {
-	pub := ecies.ImportECDSAPublic(ec.PrivKey.Public())
-	enc, err := ecies.Encrypt(rand.Reader, pub, []byte(value), nil, nil)
+	enc, err := ecies.Encrypt(rand.Reader, ec.PrivKey.Public(), []byte(value), nil, nil)
```

Comment:
Nice that we can immediately pass along the `Public` rather than doing a round about step.